### ISSUE TITLE
train.py code

### ISSUE THREAD

**Issue by augenstern-lwx** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#128]
Hi！ How can I modify the code of ‘train.py’ to get the loss of 5 stages？ Thank you

**Comment by Daniil-Osokin** [comment]
Hi! Just pass `--num_stages 5` to train a model with 5 refinement stages.
